inst/tests/tests.Rraw

```diff
@@ -8872,13 +8872,13 @@ test(1635.2, fread(text, fill=TRUE), setnames(ans1[, 1:7], c(letters[1:5], paste
 # testing function type in dt, #518
 dt = data.table(x=1, y=sum)
 test(1636.1, class(dt$y), "list")
-test(1636.2, any(grepl("1: 1 <function>", capture.output(print(dt)))), TRUE)
+test(1636.2, any(grepl("1: 1 <function\\[1\\]>", capture.output(print(dt)))), TRUE)
```

Comment:
i would just use `fixed=TRUE`

Reply:
+1 I updated these old tests to use `output=` too.